src/actions/TransactionListActions.ts

```diff
@@ -178,7 +178,10 @@ export function newTransactionsRequest(navigation: NavigationBase, walletId: str
     if (isTransactionForSelectedWallet) dispatch(fetchTransactions(walletId, selectedCurrencyCode, options))
     if (receivedTxs.length) dispatch(checkFioObtData(walletId, receivedTxs))
     if (!isReceivedTransaction(edgeTransaction)) return
-    if (spamFilterOn && !zeroString(exchangeRate) && gte(edgeTransaction.nativeAmount, calculateSpamThreshold(exchangeRate, exchangeDenom))) {
+    if (
+      !spamFilterOn ||
+      (spamFilterOn && !zeroString(exchangeRate) && gte(edgeTransaction.nativeAmount, calculateSpamThreshold(exchangeRate, exchangeDenom)))
```

Comment:
The first `!spamFilterOn ||` bit ensures that `spamFilterOn` will always be true in the second bit. Therefore, you can drop the `(spamFilterOn && ` part, since that's redundant.